docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1524,14 +1607,5 @@ Hive 3 related limitations
   file formats created by Hive 3.1 or later. When reading from these file formats,
   Trino returns different results than Hive.
 
-* :doc:`/sql/create-table-as` can be used to create transactional tables in ORC format like this::
-
-      CREATE TABLE <name>
-      WITH (
-          format='ORC',
-          transactional=true
-      )
-      AS <query>
-
-  Trino does not support gathering table statistics for Hive transactional tables.
-  You need to use Hive to gather table statistics with ``ANALYZE TABLE COMPUTE STATISTICS`` after table creation.
+* Trino does not support gathering table statistics for Hive transactional tables.
+  You must use Hive to gather table statistics with ``ANALYZE TABLE COMPUTE STATISTICS`` after table creation.
```

Comment:
Hive used to require multiple analyze runs (`ANALYZE TABLE COMPUTE STATISTIC` and `ANALYZE TABLE COMPUTE STATISTIC FOR COLUMNS`).  Maybe we don't need to document how to collect stats _on Hive_. Maybe we just let people check the Hive docs for that?

cc @m57lyra @ebyhr

Reply:
Sent #16376